package.json

```diff
@@ -50,9 +52,13 @@
     "seamless-immutable": "^7.1.2",
     "styled-components": "^2.2.0",
     "uuid": "^3.1.0",
-    "wordcloud": "^1.0.6"
+    "wordcloud": "^1.0.6",
+    "@storybook/react": "^3.2.12",
+    "@storybook/addon-actions": "^3.2.12",
+    "@storybook/addon-links": "^3.2.12"
   },
   "devDependencies": {
+    "babel-preset-es2015": "^6.24.1",
```

Comment:
don't think this is needed?

Reply:
Yeah, because without it not work storybook